CHANGELOG.md

```diff
@@ -9,6 +9,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+### Added
+
+- support `starknet_estimateFee` in the JSON-RPC v0.3 API
+
```

Comment:
Maybe we should highlight that its an extra bonus feature? Wonder if we should also highlight it in the readme 🤔 

```suggestion
- support `starknet_estimateFee` in the JSON-RPC v0.3 API
    - supports multi-transaction estimates (not sure what correct the word is here)
    - this includes declaring and immedietely deploying a class (not possible via gateway)
```